content/en/docs/contributing/docs.md

```diff
@@ -23,3 +23,137 @@ This way, changing one sentence only affects one line in the source code.
 
 Prefer using the 1st person plural.
 Use phrases like "we run the command / app", "we look at the source code", "we provide support".
+
+### Content layout
+
+The content is structured as such:
+
+```tree
+content/
+`-- en
+    |-- blog                    # Blog posts, <date>-<blogpost title>.md
+    |
+    |-- community               # Community related content.
+    |   |-- contacts.md         # Community contacts.
+    |   |-- events.md           # Informations related to reccuring and upcoming events.
+    |   |-- github.md           # Unikraft GitHub community layout.
+    |   |-- hackathons/         # Hackathon sessions, challenges and sollutions.
+    |   |-- _index.md           # Community first page.
+    |   |-- people.md           # List of people in the community.
+    |   |-- publications.md     # List of Unikraft-related publications (theses, peer-reviewed publications, etc.).
+    |   `-- talks.md            # List of Unikraft-related talks and presentations.
+    |
+    |-- docs                    # Unikraft documentation, split by categories into subdirectories.
+    |
+    |-- faq                     # List of frequently asked questions.
+    |
+    `-- releases                # Unikraft releases summaries.
```

Comment:
Add a slash (`/`) at the end of directory names. Remove dot (`.`) at the end of comments (it should only be used for sentences).